clients/rust/tests/setup/vote.rs

```diff
@@ -0,0 +1,37 @@
+#![cfg(feature = "test-sbf")]
+#![allow(dead_code)]
+
+use solana_program_test::ProgramTestContext;
+use solana_sdk::{account::Account, clock::Epoch, pubkey::Pubkey, vote::state::VoteState};
+
+pub fn create_vote_account(
+    context: &mut ProgramTestContext,
+    node: &Pubkey,
+    authority: &Pubkey,
+) -> Pubkey {
+    create_vote_account_with_program_id(context, node, authority, &solana_sdk::vote::program::ID)
+}
+
+pub fn create_vote_account_with_program_id(
+    context: &mut ProgramTestContext,
+    node: &Pubkey,
+    authority: &Pubkey,
+    program_id: &Pubkey,
+) -> Pubkey {
+    let vote = Pubkey::new_unique();
+    let mut data = vec![0u8; VoteState::size_of()];
+    // 0-4 is the version offset
+    data[4..36].copy_from_slice(node.as_ref()); // node_pubkey
+    data[36..68].copy_from_slice(authority.as_ref()); // authorized_withdrawer
```

Comment:
I'd be careful here. I know this program only cares about the first 68 bytes, but testing with accounts that don't match the real-live vote state could have unintended side effects.

Could you use a combination of `..Default::default()` and serialize?

https://github.com/anza-xyz/agave/blob/b5f5c3cdd3f9a5859c49ebc27221dc27e143d760/sdk/program/src/vote/state/mod.rs#L383-L391

Reply:
Looks much better now, thanks!